lib/Doctrine/DBAL/Connection.php

```diff
@@ -1123,73 +1123,38 @@ public function transactional(Closure $func)
      */
     public function setNestTransactionsWithSavepoints($nestTransactionsWithSavepoints)
     {
-        if ($this->_transactionNestingLevel > 0) {
-            throw ConnectionException::mayNotAlterNestedTransactionWithSavepointsInTransaction();
-        }
-
-        if ( ! $this->getDatabasePlatform()->supportsSavepoints()) {
-            throw ConnectionException::savepointsNotSupported();
-        }
-
-        $this->_nestTransactionsWithSavepoints = (bool) $nestTransactionsWithSavepoints;
+        $this->transactionManager->setNestTransactionsWithSavepoints($nestTransactionsWithSavepoints);
     }
 
     /**
      * Gets if nested transactions should use savepoints.
      *
+     * @deprecated Use getTransactionManager()->getNestTransactionsWithSavepoints()
+     *
      * @return boolean
      */
     public function getNestTransactionsWithSavepoints()
     {
-        return $this->_nestTransactionsWithSavepoints;
-    }
-
-    /**
-     * Returns the savepoint name to use for nested transactions are false if they are not supported
-     * "savepointFormat" parameter is not set
-     *
-     * @return mixed A string with the savepoint name or false.
-     */
-    protected function _getNestedTransactionSavePointName()
```

Comment:
Removing this method is a BC break. @Ocramius is it acceptable? If so it has to be documented in the `UPGRADE.md`.